components/search/testimony/TestimonyHit.tsx

```diff
@@ -44,6 +46,14 @@ const TestimonyResult = ({ hit }: { hit: Hit<Testimony> }) => {
   const { user } = useAuth()
   const { followOrg } = useFlags()
   const isCurrentUser = user?.uid === hit.authorUid
+  const isMobile = useMediaQuery("(max-width: 768px)")
+
+  function limitStringLength35Char(str: string) {
```

Comment:
nit: We added a utility function in `formatting.tsx` for this a while back - we may want to standardize on that going forward: https://github.com/codeforboston/maple/blob/cbca68809c77d2a3233ad7a6503851fe6718589a/components/formatting.tsx#L69

Reply:
refactored/done